control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go

```diff
@@ -2987,12 +2993,33 @@ func (r *HostedControlPlaneReconciler) reconcileKubeAPIServer(ctx context.Contex
 		return fmt.Errorf("failed to reconcile localhost kubeconfig secret: %w", err)
 	}
 
-	externalKubeconfigSecret := manifests.KASExternalKubeconfigSecret(hcp.Namespace, hcp.Spec.KubeConfig)
-	if _, err := createOrUpdate(ctx, r, externalKubeconfigSecret, func() error {
+	// Generate the new customExternalKubeconfig secret if the KubeAPIServerDNSName is set
```

Comment:
This seems to be using 3 different terms for the same thing: the term set(CustomKubeconfig)Status in l752, the term customExternalKubeconfig, and (KASCustomKubeconfig)Secret in l2997. Can we use one consistently everywhere?

Reply:
Another one Reconcile(ExternalKubeconfigSecret) in l3005

Reply:
👍 Thanks! I will put all of them as *KASCustomKubeconfig*. I think is the most accurate name.